Watchman.Discord/Areas/Commons/DiscordRequestExtensions.cs

```diff
@@ -50,20 +56,17 @@ private static TimeSpan ParseToTimeSpan(DiscordRequest discordRequest, TimeSpan
             {
                 throw new TimeCannotBeNegativeException();
             }
-
-            // huge value will be too big for parsing to DateTime, so I use ushort (instead of int) to be sure that the value isn't too big
-            if (timeAsNumber >= ushort.MaxValue) 
-            {
-                throw new TimeIsTooBigException();
-            }
-
             var parsedTimeSpan = lastChar switch
             {
                 's' => TimeSpan.FromSeconds(timeAsNumber),
```

Comment:
używamy tego gdzieś? 
jak tak to czy musimy?

Reply:
Używamy w MuteController i AdministrationController, więc trzeba by zamienić obie komendy na IBotCommand